tests/auth_tests/test_basic.py

```diff
@@ -53,14 +53,18 @@ def test_unicode_username(self):
 
     def test_user_no_email(self):
         "Users can be created without an email"
-        u = User.objects.create_user('testuser1')
-        self.assertEqual(u.email, '')
-
-        u2 = User.objects.create_user('testuser2', email='')
-        self.assertEqual(u2.email, '')
-
-        u3 = User.objects.create_user('testuser3', email=None)
-        self.assertEqual(u3.email, '')
+        cases = [
+            {},
+            {'email': ''},
+            {'email': None},
+        ]
+        for i, kwargs in enumerate(cases):
+            with self.subTest(**kwargs):
+                u = User.objects.create_user(
```

Comment:
This can be single-lined:
```python
user = User.objects.create_user('testuser%s' % i, **kwargs)
```

Reply:
Yes, I could be... 

I put them on separate lines deliberately. One is test boilerplate. The other doing the actual business of the test. The extra lines makes that easier to visually parse. Was the thought.

Reply:
Sure, just a suggestion.